lib/utils.js

```diff
@@ -145,6 +146,67 @@ function getActionEntryFile (pacakgejson) {
   return 'index.js'
 }
 
+/**
+ * Turns glob into an awaitable function
+ * @param {String} pattern glob pattern to match
+ * @returns {Promise}
+ */
+async function getMatchingFileList (pattern) {
+  return new Promise((resolve, reject) => {
```

Comment:
nit: personally I would use [globby](https://www.npmjs.com/package/globby) instead since it supports async-await/promises (so no need to wrap here) and supports multiple patterns